launch/launch/actions/execute_process.py

```diff
@@ -165,10 +165,11 @@ def __init__(
         :param: prefix a set of commands/arguments to preceed the cmd, used for
             things like gdb/valgrind and defaults to the LaunchConfiguration
             called 'launch-prefix'
-        :param: output configuration for process output logging. Default is 'log' i.e.
-            log both stdout and stderr to launch main log file and stderr to the screen.
-            See `launch.logging.get_output_loggers()` documentation for further reference
-            on all available options.
+        :param: output configuration for process output logging. Defaults to 'log'
+            i.e. log both stdout and stderr to launch main log file and stderr to
+            the screen. Overridden externally by the $OVERRIDE_LAUNCH_PROCESS_OUTPUT
```

Comment:
Nitpick: a new sentence should start on a newline. Same below.

Same comment about the `$` as in https://github.com/ros2/launch_ros/pull/10#discussion_r269235964

Reply:
Sure, I tend to add the `$` to emphasize it's an envvar. FIxed.